RELEASE_NOTE_DRAFT.md

```diff
@@ -6,36 +6,34 @@ This is the 9.0.0 major release.
 New Functionality
 -----------------
 
+* Implement tap on primitive iterables.
+* Implement Implement ReversibleIterable.reverseForEachWithIndex().
 * Implement distinctBy on ListIterable.
 * Implement adapt on factory classes.
 * Implement UnifiedSet.trimToSize().
 * Implement UnifiedMap.trimToSize().
-* Add new profiles for p2 releases.
 * Implement ofAll and withAll for Int, Long, Double collection factories with primitive streams.
 * Implement stream, parallelStream and spliterator on all object valued maps.
-* Sign p2 repository artifacts.
 * Implement HashingStrategies.nullSafeFromFunction(Function).
 * Implement spliterator, stream, parallelStream and castToCollection on ImmutableCollection.
 * Add an efficient default implementation of forEach.
 * Implement flatCollect on lazy primitive iterables and cartesianProduct on primitive sets factory.
 * Add covariant override for subList() in ListIterable hierarchy.
 * Implement averageIfEmpty and medianIfEmpty on primitive iterables.
 * Implement countBy and countBy with target as default methods on RichIterable.
-
-Java 9 Compatibility Fixes
---------------------------
-
 * For Java 9 add "Automatic-Module-Name" to manifests of JAR files.
-* Add a Java 9 regression build.
 
 Tech Debt Reduction
 -------------------
 
-* Fix Travis build, remove Java 9 build, remove Sonar badge.
+* Deprecate or hide collection factory constructors. 
```

Comment:
Should this be under breaking change? I do not know, so asking.

Reply:
I assumed that was what your original comment meant - Line 42: This is a breaking change

Reply:
Ok fixing now.